### PR TITLE
Adds the odo website to the master branch

### PR DESCRIPTION
**Pull request by cdrage**
This adds the odo website to the master branch of odo.

From here on out, we'll modify any site changes (layout and such) in the
root directory, while the `gh-pages` branch will contain the built / raw
html.

### CODE REVIEW

**Comment by cdrage** [comment]
This is a step in the right direction @rkratky

If someone could merge this ASAP that'd be great. After it's merged I can update the synchronization script.

File: site/pkg/build/.coverprofile [hunk 1]
```diff
@@ -0,0 +1,89 @@
+mode: atomic
+github.com/kedgeproject/kedge/pkg/build/build.go:43.79,46.16 2 0
```
Comment (surajnarwade) [inline site/pkg/build/.coverprofile]: kedge :question:

Comment (kadel) [reply site/pkg/build/.coverprofile]: good catch.
This file shouldn't be there. It is unit test coverage

Comment (cdrage) [reply site/pkg/build/.coverprofile]: wow that's an old file. thanks! removed.

**Comment by kadel** [comment]
```
level=fatal msg="waiting for Kubernetes API: context deadline exceeded"
```
/retest

**Pull request merged**